src/regex.cr

```diff
@@ -257,6 +257,23 @@ class Regex
     #
     # Unsupported with PCRE.
     ENDANCHORED = 0x8000_0000
+
+    # Do not check the pattern for valid UTF encoding.
+    NO_UTF_CHECK = NO_UTF8_CHECK
+
+    # Enable matching against subjects containing invalid UTF bytes.
+    # Invalid bytes never match anything. The entire subject string is
+    # effectively split into segments of valid UTF.
+    #
+    # Read more in the [PCRE2 documentation](https://www.pcre.org/current/doc/html/pcre2unicode.html#matchinvalid).
+    #
+    # When this option is set, `MatchOptions::NO_UTF_CHECK` is ignored at match time.
+    #
+    # Unsupported with PCRE.
+    #
+    # NOTE: This option was introduced in PCRE2 10.34 but a bug that can lead to an
```

Comment:
That deserves a stronger marker IMO.

```suggestion
    # WARNING: This option was introduced in PCRE2 10.34 but a bug that can lead to an
```

Reply:
I think it's fine. If you want to use a feature of any library you must make sure it's supported in the library version you're linking against. That's a general axiom. Here we're just pointing out the relevant versions for this particular option.

Reply:
We're pointing not only the version but also the fact it contains a rather serious bug.